Tests/conf.json

```diff
@@ -3163,6 +3163,11 @@
             "integrations": "Public DNS Feed",
             "playbookID": "Public_DNS_Feed_Test",
             "fromversion": "5.5.0"
+        },
+        {
+            "integrations": "BitcoinAbuse",
+            "playbookID": "BitcoinAbuse-test",
+            "fromversion": "5.0.0"
```

Comment:
i changed fromversion to 5.5.0 because it seemed the build of 5.0. tried to run the test playbook, but it didnt work because feeds are from 5.5.0

then having fromversion here makes sense?